Tanasee/Tanasee1.xml

```diff
@@ -280,52 +279,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <decoNote xml:id="d1" type="frame">
                                 <locus target="#6r"/>
                                 <desc>Finely illuminated frame as described and reproduced in <bibl><ptr target="bm:Heldman1972miniatures"/>
-                                    <citedRange>Figure 85</citedRange></bibl> surrounding the text of the <ref target="LIT1349EpistlEusebius"/> 
-                                    as described in <ref target="#ms_i1"/></desc>
+                                    <citedRange>Figure 85</citedRange></bibl> surrounding the text of the 
+                                    <ref type="work" target="LIT1349EpistlEusebius"/> as described in <ref target="#ms_i1"/></desc>
                             </decoNote>
                             <decoNote xml:id="d2" type="miniature">
                                 <locus target="#6v"/>
                                 <desc>Finely illuminated frame, that was described and reproduced in 
                                     <bibl><ptr target="bm:LeroyAeth1968"/><citedRange>Abb. 430a</citedRange></bibl> and
                                     <bibl><ptr target="bm:Heldman1972miniatures"/><citedRange>Figure 86</citedRange></bibl> 
-                                    surrounding the text of the <ref target="LIT1349EpistlEusebius"/> in <ref target="#ms_i1"/>.</desc>
+                                    surrounding the text of the <ref type="work" target="LIT1349EpistlEusebius"/> in <ref target="#ms_i1"/>.</desc>
                             </decoNote>
                             <decoNote xml:id="d3" type="miniature">
                                 <locus target="#7r"/>
                                 <desc>Finely illuminated frame, that was described and reproduced in 
                                     <bibl><ptr target="bm:Leroy1962Recherches"/><citedRange>Figure 7</citedRange></bibl>, surrounding 
-                                    the text of <ref target="LIT1349EpistlEusebius"/> as described in <ref target="#ms_i1"/>.</desc>
+                                    the text of <ref type="work" target="LIT1349EpistlEusebius"/> as described in <ref target="#ms_i1"/>.</desc>
                             </decoNote>
                             <decoNote xml:id="d4" type="frame">
                                 <locus target="#7v"/>
-                                <desc>Finely illuminated frame surrounding <ref target="LIT1224Canons"/> as described in <ref target="#ms_i2"/>.</desc>
+                                <desc>Finely illuminated frame surrounding <ref type="work" target="LIT1224Canons"/> as described in 
+                                    <ref target="#ms_i2"/>.</desc>
```

Comment:
sorry, missed. ref to an item in the record should have the `type="item"` here and elsewhere

Reply:
in general, check that `ref `has `type`

Reply:
OK. I went through all the ref in decoDesk and put "ref" in a search. I hope, that nothing has escaped my attention. If I understand correctly every ref needs a specification. So, it must also be 
`<ref type="item" target="#a42"/>`
instead of `<ref target="#a42"/>` and the like.

Reply:
not every `ref`, `item `is for `msItem `only 

https://betamasaheft.eu/Guidelines/?id=ref
https://betamasaheft.eu/Guidelines/?id=summary-on-the-use-of-ref-and-corresp

Reply:
Ok. If I understand it correctly, references to additions need to stay like this without a type: 
`<ref target="#a42"/> `

Reply:
yes )

Reply:
OK. Done. I eliminated type in references to additions, but I added or kept type=authFile for authority files and type=deco for decorations.